docs/features/logforwarding/outputs/syslog-forwarding.adoc

```diff
@@ -0,0 +1,149 @@
+      payloadKey: message # OPTIONAL parameter can be skipped will be calculated automatically <5>
+      appName: myApp # OPTIONAL parameter can be skipped will be calculated automatically <6>
+      procId: 3144 # OPTIONAL parameter can be skipped will be calculated automatically <7>
+      msgId: MsgID # OPTIONAL parameter can be skipped will be calculated automatically <8>
+      enrichment: KubernetesMinimal # <9>
+    type: syslog
+  pipelines:
+    - name: demo-logs
+      inputRefs:
+        - application
+        - infrastructure
+        - audit
+      outputRefs:
+        - syslog
+  serviceAccount:
+    name: mine
+----
+Where `url` and `rfc` parameters are mandatory other are optional
+
+. `rfc`: Syslog RFC specification:  `RFC3164` or `RFC5424`
+. `url`: an absolute URL, with a scheme. Valid schemes are: `http`, `https`, `tcp`, `tls`, `udp` and `udps`
+. `severity`: values are defined in https://tools.ietf.org/html/rfc5424#section-6.2.1
+  The value can be a decimal integer or one of these case-insensitive keywords:
+
+     Emergency
+     Alert
+     Critical
+     Error
+     Warning
+     Notice
+     Informational
+     Debug
+
+. `facility`: values are defined in https://tools.ietf.org/html/rfc5424#section-6.2.1.
+  The value can be a decimal integer. Facility keywords are not standardized,
+  this API recognizes at least the following case-insensitive keywords
+  (defined by https://en.wikipedia.org/wiki/Syslog#Facility_Levels):
+
+    kernel user mail daemon auth syslog lpr news
+    uucp cron authpriv ftp ntp security console solaris-cron
+    local0 local1 local2 local3 local4 local5 local6 local7
+
+
+. `payloadKey`: specifies record field to use as payload. This supports template syntax to allow dynamic per-event values.
+  The `payloadKey` must be a single field path encased in single curly brackets `{}`.
+  Field paths must only contain alphanumeric and underscores. Any field with other characters must be quoted.
+  If left empty, Syslog will use the whole message as the payload key.
+  Examples:
+
+  {.bar}
+  {.foo.bar.baz}
+  {.foo.bar."baz/with/slashes"}
+
+. `appName`: is APP-NAME part of the syslog-msg header.
+  `appName` needs to be specified if using `rfc5424`. The maximum length of the final values is truncated to 48. This supports template syntax* to allow dynamic per-event values.
+
+.  `procId`: is `PROCID` part of the syslog-msg header.
+   `procId` needs to be specified if using `rfc5424`. The maximum length of the final values is truncated to 128. This supports template syntax* to allow dynamic per-event values.
+
+.  `msgId`: is `MSGID` part of the syslog-msg header. This supports template syntax to allow dynamic per-event values. MsgId needs to be specified if using `rfc5424`.  The maximum length of the final values is truncated to 32. This supports template syntax* to allow dynamic per-event values.
+
+. `enrichment`: `None` or `KubernetesMinimal`.
```

Comment:
Add note this applies only to container logs.  Consider working it as "application logs" and/or "infrastructure container" logs